cmd/gaia/cli_test/test_helpers.go

```diff
@@ -163,7 +163,7 @@ func (f *Fixtures) Flags() string {
 
 // UnsafeResetAll is gaiad unsafe-reset-all
 func (f *Fixtures) UnsafeResetAll(flags ...string) {
-	cmd := fmt.Sprintf("gaiad --home=%s unsafe-reset-all", f.GDHome)
+	cmd := fmt.Sprintf("../../../build/gaiad --home=%s unsafe-reset-all", f.GDHome)
```

Comment:
I feel like this is something we should define on the `Fixtures` object (maybe `gdcmd` and `gclicmd` attributes) then it can be easily switched in and out for different usages (one example would be checking in `NewFixtures` if `CI=true` and directing to a previously built binary).